cmd/up.go

```diff
@@ -54,6 +54,8 @@ type UpCmd struct {
 
 	DotfilesSource string
 	DotfilesScript string
+
+	gitIgnore bool
```

Comment:
This should be a flag eventually, right? Could you add it to the `upCmd` flags in line 119 and following please?